rllib/algorithms/algorithm.py

```diff
@@ -31,9 +31,9 @@
 import tree  # pip install dm_tree
 
 import ray
-from ray.air.constants import TRAINING_ITERATION
 from ray._private.usage.usage_lib import TagKey, record_extra_usage_tag
 from ray.actor import ActorHandle
+from ray.air.constants import TRAINING_ITERATION
```

Comment:
Can we import this from `ray.tune.result` instead, together with the `TIME_TOTAL_S`?

Reply:
done